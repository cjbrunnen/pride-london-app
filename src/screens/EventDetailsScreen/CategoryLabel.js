// @flow
import React from "react";
import { StyleSheet } from "react-native";
import Text from "../../components/Text";
import {
  eventCategoryLabelBgColor,
  eventDetailsBgColor
} from "../../constants/colors";

type Props = {
  categoryName: string
};

const CategoryLabel = ({ categoryName }: Props) => (
  <Text style={styles.categoryLabel} type="h3">
    {categoryName}
  </Text>
);

const styles = StyleSheet.create({
  categoryLabel: {
    color: eventDetailsBgColor,
    backgroundColor: eventCategoryLabelBgColor,
    paddingHorizontal: 8,
    marginRight: 8,
    marginBottom: 8
  }
});

export default CategoryLabel;
