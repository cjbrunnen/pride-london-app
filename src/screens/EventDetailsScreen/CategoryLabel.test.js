// @flow
import React from "react";
import { shallow } from "enzyme";
import CategoryLabel from "./CategoryLabel";

it("renders correctly", () => {
  const output = shallow(<CategoryLabel categoryName="music" />);
  expect(output).toMatchSnapshot();
});
