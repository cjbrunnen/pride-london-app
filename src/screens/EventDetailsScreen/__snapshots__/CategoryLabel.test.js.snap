// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  markdown={false}
  style={
    Object {
      "backgroundColor": "#D8D8D8",
      "color": "#FFF",
      "marginBottom": 8,
      "marginRight": 8,
      "paddingHorizontal": 8,
    }
  }
  type="h3"
>
  music
</Text>
`;
